crud-rest-service/src/users/dto/create-user.dto.ts

```diff
@@ -0,0 +1,38 @@
+import {
+  IsString,
+  IsNumber,
+  IsBoolean,
+  IsNotEmpty,
+  Matches,
+  Max,
+  Min,
+} from 'class-validator';
+
+const passwordPattern = /^(?=.*[0-9])(?=.*[a-zA-Z])([a-zA-Z0-9]+)$/;
+export class CreateUserDto {
+  @IsString()
+  @IsNotEmpty()
+  readonly login: string;
+
+  @IsString()
+  @IsNotEmpty()
+  readonly email: string;
```

Comment:
The same, User type/interface don't have email field , why we should use it?

![image](https://user-images.githubusercontent.com/17615969/180645219-79f0c74d-2047-46de-8a39-df1143b2ca75.png)

Reply:
At first I added the address by accident, but then I decided to leave it.
But ok, I can delete )